rose/server/web/settings/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Set Drivers</title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+<body class="dark-theme">
+    <h2 class="heading">Set Game Drivers</h2>
+    <settings-form class="form" />
```

Comment:
custom-elements can't be void
```suggestion
    <settings-form class="form"></settings-form>
```

Reply:
wow, fixed, btw/ why did it work ?

Reply:
I see it was a hot topic :fire: 
https://github.com/WICG/webcomponents/issues/624